app/models/user.rb

```diff
@@ -80,4 +80,16 @@ def remove_pregnancy(pregnancy)
     pregnancies.delete pregnancy
     reload
   end
+
+  def clear_call_list
+    pregnancies.each do |p|
+      pregnancies.delete(p) if recently_reached?(p)
+    end
+  end
+
+  private
+
+  def recently_reached?(preg)
```

Comment:
That would be fine, but I was modeling it after recently_called? so I would want to change both to maintain consistency. 

To that effect, do you think these should be public or private methods? They should probably both be one or the other.